go/enclave/enclave.go

```diff
@@ -295,6 +296,15 @@ func (e *enclaveImpl) GetBatchBySeqNo(seqNo uint64) (*common.ExtBatch, common.Sy
 	return b, nil
 }
 
+func (e *enclaveImpl) GetRollupData(internalRollup []byte) (*big.Int, *uint64, common.SystemError) {
+	calldataRollupHeader := new(common.CalldataRollupHeader)
```

Comment:
this is perfect stuff to retrieve from the database.
They should already be written in the "rollup" table.
It's a matter of selecting them from there

Reply:
Done